File: CS50’s Introduction to Computer Science/Week01_C/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

void printPyramids(int height);
int evaluateHeight();

int main(void)
{
    printPyramids(evaluateHeight());
    return 0;
}

int evaluateHeight()
{
    int height;
   do{
        height = get_int("Height: ");

    }while(height<1 || height >8);

    return height;
}

void printPyramids(int height)
{
    int space = height-1;
    for(int i=0; i<height; i++)
    {
         for(int k=0; k<space; k++)
        {
            printf(" ");

        }
        for(int j=0; j<=i; j++)
        {

            printf("#");
        }

        printf("  ");

        for(int j=0; j<=i; j++)
        {

            printf("#");
        }

        space--;

        printf("\n");
    }
}

File: Introduction to Computer Science/Week06_Python/mario.py
from cs50 import get_int
height = 0
while (height <= 0 or height >=9):
    height = get_int("Height: ")

spaces = height-1
for i in range(height):
    for j in range(spaces):
        print(" ", end="")
    for k in range(i+1):
        print("#", end="")
    print(" ", end="")
    for l in range(i+1):
        print("#", end="")
    spaces -= 1
    print()
